tests/integrationtest/r/planner/core/indexmerge_path.result

```diff
@@ -1081,3 +1081,127 @@ IndexMerge	0.05	root		type: union
 └─Selection(Probe)	0.05	cop[tikv]		eq(planner__core__indexmerge_path.t.a, 1)
   └─TableRowIDScan	49.94	cop[tikv]	table:t	keep order:false, stats:pseudo
 SET @@tidb_opt_fix_control = default;
+drop table if exists t, t1;
```

Comment:
the code change makes sense to me, by the way, is there any test case that needs to be modified?